voxblox/include/voxblox/integrator/merge_integration.h

```diff
@@ -90,16 +91,16 @@ void mergeLayerAintoLayerB(const Layer<VoxelType>& layer_A,
 // transformLayer for details
 template <typename VoxelType>
 void mergeLayerAintoLayerB(const Layer<VoxelType>& layer_A,
-                           const Transformation& T_A_B,
+                           const Transformation& T_B_A,
```

Comment:
these changes in the transformation notation were reviewed and discussed in this PR: https://github.com/ethz-asl/voxblox/pull/150